samples/integration/pom.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.experimental</groupId>
+		<artifactId>spring-native-sample-parent</artifactId>
+		<version>0.10.0-SNAPSHOT</version>
+		<relativePath>../maven-parent/pom.xml</relativePath>
+	</parent>
+	<groupId>com.example</groupId>
+	<artifactId>integration</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<name>integration</name>
+	<description>GraalVM Spring Integration Demo</description>
+
+	<properties>
+		<java.version>11</java.version>
+		<spring-integration.version>5.5.0-SNAPSHOT</spring-integration.version>
```

Comment:
I need this version explicitly because there were some changes in SI already after an RC

Reply:
When next RC/release is planned?

Reply:
May 18th, just before Spring Boot 2.5 GA: https://github.com/spring-projects/spring-integration/milestone/63 !